packages/workbox-build/src/entry-points/generate-sw.js

```diff
@@ -17,13 +17,16 @@
 const path = require('path');
 
 const cdnUtils = require('../lib/cdn-utils');
-const copyWorkboxSW = require('../lib/copy-workbox-sw');
+const copyWorkboxLibraries = require('../lib/copy-workbox-libraries');
 const generateSWSchema = require('./options/generate-sw-schema');
 const getFileManifestEntries = require('../lib/get-file-manifest-entries');
 const validate = require('./options/validate');
 const writeServiceWorkerUsingDefaultTemplate =
   require('../lib/write-sw-using-default-template');
 
+// TODO: Update for https://github.com/GoogleChrome/workbox/issues/969.
+const WORKBOX_SW_FILENAME = 'workbox-sw.prod.js';
```

Comment:
Why does this need to be hardcoded? Code it use the file name in the workbox-sw package.json browser (soon to be main) field?

Reply:
Good point.